### DISCUSSION TITLE
Comments and linebreaks processing, is there a way to disable them?

### DISCUSSION THREAD

**Discussion by agonzalo36r** [estruyf/vscode-front-matter#168] [Q&A] [answered]
When I use frontmatter, it deletes all the comments in my file. Is there a way to disable it? Sometimes I need to comment one line inside the frontmatter when testing new metadata and auto-deleting kinda kill that flow. 
Also, for some platform specific RSS, I need to have  linebreaks inside the contentEncoded tag, but Front Matter also deletes those. I was reading the documentation looking for some toggle to avoid this behaviour but didn't find anything. 
Can you help me with that please? 

Thanks a lot for reading this!

**Comment by estruyf** [comment, marked as answer]
Hi, @agonzalo36r this probably has to do with the parser. I'll check if we can change its behavior.

Would you be able to share a sample of the line breaks?

**Reply by estruyf** [reply]
@agonzalo36r looked into it, and it will be a bit hard to implement, as that would mean to rewrite the parser as it ignores the commented metadata.

A trick that you could do, is to use any other prefix for the field that you want to test. Like adding a `.` as the prefix. That way, the field is not commented out for the parser and will be kept in place, your SSG will ignore it, as it doesn't know the field.

Still interested in the issue with the line breaks to see what you mean, or how you are using them. As I tested it out with some fields, like the description fields, and that seems to be working fine.

**Reply by agonzalo36r** [reply]
Thanks for the reply! And the prefix works great, thanks for that :)

About the line breaks, the thing is that I'm using some markdown on my frontmatter, specifically for the contentEncoded tag, it looks like this:

```
contentEncoded: "Paragraph 1

Paragraph 2

- List item 2

- List item 2"

```
That's because I use to generate the RSS for a Spotify podcast, and if you don't add the linebreaks, all the text appears in one line. I'm using Hugo with Goldmark, and tried using `\` to declare the line breaks, but it doesn't work, so my way to make it work was adding the line breaks in a explicit way. But besides that, Frontmatter helps A LOT. Thanks!

**Reply by estruyf** [reply]
@agonzalo36r thanks! This is actually how YAML parses your content. In order to keep the line breaks, you should not be using the quote, you need to tell YAML your using a content block / multiline. You can do this with the following character: `|`.

In your case, you should define your `contentEncoded` property as follows:

```yaml
contentEncoded: |
  Paragraph 1

  Paragraph 2

  - List item 2

  - List item 2
```

**Important**: Also keep the line indents in mind.

**Reply by estruyf** [reply]
Here you can experiment with the multiline syntax of YAML: https://yaml-multiline.info/

**Reply by agonzalo36r** [reply]
Heey, thanks a lot for that! That solved everything haha. I was really interested in the template functionality but the linebreaks and the comments were containing me from using it. Thanks for this great CMS!

**Reply by estruyf** [reply]
Thank you @agonzalo36r for using it! 🙏
